src/components/layout/OrgLayout.tsx

```diff
@@ -13,49 +13,69 @@ import OrgHeader from './OrgHeader';
 
 interface OrgLayoutProps {
     children: JSX.Element;
+    mainPage: boolean;
     orgId: string;
 }
 
-const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
+const OrgLayout = ({ children, mainPage, orgId } : OrgLayoutProps) : JSX.Element => {
     const orgQuery = useQuery(['org', orgId], getOrg(orgId));
     const router = useRouter();
     const path =  router.pathname.split('/');
     let currentTab = path.pop();
 
-    if (currentTab === '[orgId]') {
-        currentTab = 'home';
-    }
+    if (mainPage) {
```

Comment:
Instead of wrapping such a huge portion of the functionality in a conditional, we should use composition. I.e. create a separate component for "main pages", e.g. `OrgHomeLayout` and utilize composition the same way that we do for `DefaultLayout` and `OrgLayout` today.